### ISSUE TITLE
CLI Wallet: List transaction history

### ISSUE THREAD

**Issue by spikeekips** [bosnet/sebak#2]
Like 
```sh
$ sebak wallet history
```
will list the transaction and it's operations history.

**Comment by Geod24** [comment]
Given that there is an HTTP interface for this, do we still want to expose it via CLI ?

**Comment by outsideris** [comment]
I think Most features of CLI will use HTTP API.
Why do you think it doesn't need to expose it?

**Comment by spikeekips** [comment]
I think our cli command should provide the **basic** features like transaction history.

**Comment by Geod24** [comment]
Yep, that comment date back from the time I thought we'd had a separation.
It makes sense 👍
